components/inlinealert/stories/inlinealert.stories.js

```diff
@@ -61,15 +62,95 @@ export default {
 	},
 };
 
+/**
+ * The neutral variant is the default for in-line alerts.
+ */
 export const Default = InlineAlertGroup.bind({});
 Default.args = {};
 
+// ********* DOCS ONLY ********* //
+/**
+ * The informative variant uses the informative semantic color (blue) and has an "information" icon to help those with color vision deficiency discern the message tone. This should be used when the message needs to call extra attention, as compared to the neutral variant.
+ * 
+ * _Spectrum for Adobe Express uses a different icon. Use the SX_Info_18_S.svg icon in the Express workflow icon set._
+ */
+export const Informative = Template.bind({});
+Informative.args = {
+	variant: "info",
+	headerText: "in-line alert header",
```

Comment:
Is the template forcing the variant name to always sit in front of the headerText? That doesn't seem ideal if I want to see what my production headline will look like in the component but that metadata is hardcoded in.  Would you be able to correct that in this PR or does that seem too out of scope for what you're doing here?